src/app-layout/notifications/index.tsx

```diff
@@ -42,22 +41,12 @@ interface NotificationWrapperProps extends NotificationsProps {
 }
 
 export const Notifications = React.forwardRef(
-  (
-    { navigationPadding, toolsPadding, sticky, isMobile, ...rest }: NotificationWrapperProps,
-    ref: React.Ref<HTMLDivElement>
-  ) => {
+  ({ sticky, isMobile, ...rest }: NotificationWrapperProps, ref: React.Ref<HTMLDivElement>) => {
     const notificationsProps: NotificationsProps = { isMobile, ...rest };
-    return (
```

Comment:
Remove the div because all classes there for VR were removed from style.css in past PRs.